docs/webformsasinput.md

```diff
@@ -0,0 +1,156 @@
+# Webforms as input method for Archipelago Digital Objects (ADO)
+
+Drupal 8/9 provides a lot of out-of-the-box functionality to setup the way Content Entities (Nodes or in our case ADOs) are exposed to users with the proper credentials. That functionality lives under the "Display Modes" and can be accessed at `admin/structure/display-modes`.
+
+![Display Modes](../imgs/display-modes.jpg)
+
+In a few quick words, The Display Mode Concept covers: formatting your Content Entities and their associated Fields so when a user lands on a Content Page, they are displayed in a certain, hopefuly pleasing, way and also how users with proper Credentials can fill inputs/edit values for each `field` a Content Entity provides.
+
+First, formatting output (basically building the front facing page for each content entity) is done by a `View Mode`. Second, defining how/what input method you are going to use to create or edit Content entities, is handlede by a `Form Mode`. Both Modes, are, in Drupal Lingo, Configuration Entities, they provide things you can configure, you can name them and reuse them and those configurations can all be exported and reimported using YAML files. Also both Modes the following in common:
+
+- Drupal always provides a "default" one that can not be deleted.
+- You can create new ones.
+- You can apply permissions to them.
+- All Modes work on "fields", means the tiny little input/output pieces that are either part of a Content Entity or attached to them (the title, the Body, and in our case a Strawberryfield (SBF),
+- They Provide Config/setting options for each Field.
+- They are always associated to Content Types/Bundles. Means all Nodes of the same Content type will share the same modes.
+
+The main difference, other than their purpose (Output v/s Input) is that, on View Modes, the settings you apply to each field are associated to "Formatters" and on Form Modes, the settings you apply to each field are connected to "Widgets".
+
+So, resuming, what lives under the Concept of a "Display Mode": 
```

Comment:
Very picky. This would be correct, but anybody would understand as it is now.

So, resuming, what lives under the Concept of a "Display Mode": 
```suggestion
So, resuming, this is what lives under the Concept of a "Display Mode": 
```